tests/asahirov/test_homework_2.py

```diff
@@ -0,0 +1,56 @@
+"""Дз:
+ Написати тести які б перевіряли різні помилки в пайтоні наприклад:
+5 / 0
+assert False
+list()[1000]
+dict()["abc"]
+int.foo
+
+
+2. Спробувати покрити тестами ключові слова, наприклад:
+not, and, or, in , is,  not in, is not
+if elif else
+for in
+while
+def
+компрехеншни,
+і тд."""
+import pytest
+
+
+def test_exception_zero_division():
+    with pytest.raises(ZeroDivisionError):
+        res = 999/0
+
+
+def test_exception_assert():
+    with pytest.raises(AssertionError):
+        assert "soft" in "serve"
+
+
+def test_exception_index():
+    with pytest.raises(IndexError):
+        example = [_*3 for _ in range(16)]
```

Comment:
do not name reference as _    if you use it
spaces around operator

``` 
example = [item * 3 for item in range(16)]

```

Reply:
updated